File: online-judge-solutions/Codeforces/1712B.py
#https://codeforces.com/problemset/problem/1712/B
#Lang		:	Python 3.8
#Time		:	171 ms
#Memory         :	3400 KB
for _ in range(int(input())):
    n=int(input())
    v=[i+1 for i in range(n)]
    for i in range(n-1,0,-2):
        v[i],v[i-1]=v[i-1],v[i]
    print(*v)

File: online-judge-solutions/Codeforces/1712B.cpp
//https://codeforces.com/problemset/problem/1712/B
//Lang		:	GNU C++14
//Time		:	15 ms
//Memory	:	400 KB
#include<iostream>
#include<numeric>
#include<vector>
using namespace std;
int main(){
	int t,n;
	cin>>t;
	while(t--){
		cin>>n;
		vector<int> v(n);
		iota(begin(v),end(v),1);
		for(int i=n-1;i>0;i-=2){
			swap(v[i],v[i-1]);
		}
		for(int x:v){cout<<x<<" ";}cout<<endl;
	}
	return 0;
}
